src/components/NoteCard.jsx

```diff
@@ -21,7 +36,7 @@ const NoteCard = ({ notes }) => {
     try {
```

Comment:
might as well add catch block

Reply:
I think the catch block follows on 45 line.

Reply:
Yes try block is followed by catch on line 45